src/pamela/utils.clj

```diff
@@ -56,6 +80,7 @@
                             (if (fs/absolute? filename)
                               (fs/file filename)
                               (fs/file (get-cwd) filename)))
+          data (encode-ir-tokens-as-strings data)
```

Comment:
`output-file` is used to generate both `edn` and `json`.  
We shouldn't be doing the string encoding when the `file-format` is JSON.

Reply:
Yes, that was a serious error, good catch.  I have fixed it to only encode when writing JSON. I have checked it in to this branch.